src/components/dashboard/subsettings/TokenSettings.tsx

```diff
@@ -73,35 +77,78 @@ export default function TokenSettings({ id }: Props) {
     });
   }
 
+  const refreshToken = async () => {
+    const defaultToken = defaultAoTokens.find((t) => t.processId === token.id);
```

Comment:
Why refreshToken is done for default tokens only ? Why not all ao tokens present ?

Reply:
I'm not opposed to that - I just did it this way since it's on an individual token page

Reply:
@nicholaswma Yes, its on an individual token page but its not refreshing for tokens that are not the `defaultTokens`. For `!defaultToken` the refresh is not handled.

Reply:
good catch - I meant it to be the other way around, my intention was for default tokens to not have to be refreshed (this is essentially to not querying the ao token process)